kadmin/init.c

```diff
@@ -246,6 +252,15 @@ init(struct init_options *opt, int argc, char **argv)
 			    KRB5_KDB_DISALLOW_ALL_TIX, CRE_DUP_OK);
 	krb5_free_principal(context, princ);
 
+	/* Create `WELLKNONW/org.h5l.gss-preauth-state@WELLKNOWN:ORG.H5L' for preauth state */
```

Comment:
This principal is used to encrypt acceptor state, so the acceptor is stateless. We could just use the FAST cookie principal, given they have separate key usage numbers?